### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by JacksonElia**

### CODE REVIEW

**Comment by JacksonElia** [comment]
bro I thought it would let me approve my own pr lol

File: README.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ source venv/bin/activate
 ```pip install .[dev]```
 
 #### Install and start the pigpio daemon on the Raspberry Pi:
+*Every time the pi boots up, you must run this in order for the servo to work
```
Comment (harshil21) [inline README.md:27]:
```suggestion
Every time the pi boots up, you must run this in order for the servo to work
```
we should probably just add that to .bashrc

**Review by harshil21 — COMMENTED** [review]
one typo

File: README.md (hunk 1 above)
Comment (JacksonElia) [reply README.md:27]: yeah we should add it to the bashrc, but I meant to have the *

**Review by harshil21 — APPROVED** [review]

**Pull request merged**